components/gallery/ModalGallery.tsx

```diff
@@ -20,7 +29,7 @@ const Modal = ({ onClose, public_id }) => {
                     <a style={{position: "absolute", top: 10, right: 10, fontSize: 32, width: 32, height: 32}} href="#" onClick={handleCloseClick}>
                         x
                     </a>
-                    <div className="modal-body" >
+                    <div className="modal-body" id="scroll-container2" >
```

Comment:
The  modal nor its contents scroll, the `scroll-container` div in HorizontalGallery is what scrolls.

Reply:
Good callout. Though I shall also say this is leftover from a test I was doing - a very wrong test lol